x-pack/plugins/infra/public/index.ts

```diff
@@ -30,3 +30,4 @@ export type InfraAppId = 'logs' | 'metrics';
 
 // Shared components
 export { LazyLogStreamWrapper as LogStream } from './components/log_stream/lazy_log_stream_wrapper';
+export { LogStreamProps } from './components/log_stream';
```

Comment:
@afgomez 👋 This is the only infra change in the PR, which I mentioned needing earlier in Slack. You're totally welcome to check out the rest of the PR/convo as well if you're interested, but also feel free to skip/don't feel like you have to!